experimental/delegate/CMakeLists.txt

```diff
@@ -64,9 +64,11 @@ iree_lit_test_suite(
     "mlp.mlir"
     "matmul.mlir"
     "opt.mlir"
+    "tres-leches.mlir"
```

Comment:
@daveliddell  Does this file need to be checked in?

Reply:
Absolutely it does!  (Eheheh)